src/util/log.hpp

```diff
@@ -7,7 +7,7 @@
 #define LOG_CALLBACK
 //#define LOG_PRINT_LOC
 #define LOG_PRINT_COLORS
-//#define LOG_PRINT_TO_OUTPUT
+#define LOG_PRINT_TO_OUTPUT
```

Comment:
Revert this, Appears to be broken. I've created an issue for it.

Reply:
Reverted it.